src/modify.js

```diff
@@ -66,12 +67,39 @@ function rewriteAllFields(schema, configCallback, context) {
   });
 }
 
+function setFields(schema, fieldsConfig) {
+  if (!fieldsConfig) return null;
+
+  const fieldsToAdd = Object.entries(fieldsConfig);
+
+  fieldsToAdd.forEach(([shortPath, fieldAttrs]) => {
+    const fieldPath = shortToFullPath(shortPath);
+
+    if (fieldAttrs.properties) {
+      // Recursive to nested fields...
+      setFields(get(schema.properties, fieldPath), fieldAttrs.properties);
+    }
+
+    const fieldInSchema = get(schema.properties, fieldPath);
+    if (fieldInSchema) {
+      // NOTE: Not sure if we should ignore or overwrite it
```

Comment:
Note on why I decided to ignore modifying existing fields: Read this comment first https://github.com/remoteoss/json-schema-form/pull/80/files#r1683547346.

So if `fields` is exclusive for existing fields, then `create` is exclusive for new fields. Does it make sense?